tests/baselines/reference/parser.numericSeparators.binaryNegative.errors.txt

```diff
@@ -0,0 +1,62 @@
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/1.ts(1,5): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/2.ts(1,3): error TS1177: Binary digit expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/2.ts(1,3): error TS2304: Cannot find name '_110'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts(1,2): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts(1,3): error TS1005: ';' expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts(1,3): error TS2304: Cannot find name 'B0101'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts(1,5): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts(1,6): error TS1005: ';' expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts(1,6): error TS2304: Cannot find name '_11'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/5.ts(1,12): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/5.ts(1,13): error TS1005: ';' expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/5.ts(1,13): error TS2304: Cannot find name '_'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/6.ts(1,3): error TS1177: Binary digit expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/6.ts(1,3): error TS2304: Cannot find name '___0111010_0101_1'.
+
+
+==== tests/cases/conformance/parser/ecmascriptnext/numericSeparators/1.ts (1 errors) ====
+    0b00_
+        ~
+!!! error TS6188: Numeric seperators are not allowed at the end of a literal.
+    
+==== tests/cases/conformance/parser/ecmascriptnext/numericSeparators/2.ts (2 errors) ====
+    0b_110
+      
+!!! error TS1177: Binary digit expected.
+      ~~~~
+!!! error TS2304: Cannot find name '_110'.
+    
+==== tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts (3 errors) ====
+    0_B0101
+     ~
+!!! error TS6188: Numeric seperators are not allowed at the end of a literal.
+      ~~~~~
+!!! error TS1005: ';' expected.
+      ~~~~~
+!!! error TS2304: Cannot find name 'B0101'.
+    
+==== tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts (3 errors) ====
+    0b01__11

```

Comment:
As above, after we report the numeric separator error, we may want to continue to parse out the rest of the number to avoid excessive unrelated errors, though I'd like to get @mhegazy's or @DanielRosenwasser's opinion on that.